ffc/representation.py

```diff
@@ -192,6 +192,54 @@ def _compute_element_ir(ufl_element, element_numbers, classnames, parameters, ji
     return ir
 
 
+def _compute_dofmap_permutation_tables(fiat_element, cell):
+    """Create tables of edge permutations and facet permutations for all the possible
+    orientations of the cell."""
+    if isinstance(fiat_element, MixedElement):
+        elems = fiat_element.elements()
+    else:
+        elems = (fiat_element, )
+
+    td = cell.topological_dimension()
+
+    # Lists of permutations for each edge or facet (if any)
+    edge_permutations = []
+    face_permutations = []
+
+    offset = 0
+    for el in elems:
```

Comment:
Can we use more readable variable names? Could this be `for element in elements:`?